config/config.example.toml

```diff
@@ -43,6 +43,7 @@ pool_size = 5              # Number of connections to keep open
 reconnect_max_attempts = 5 # Maximum number of reconnection attempts to make before failing. Set to 0 to retry forever.
 reconnect_delay = 5        # Delay between reconnection attempts, in milliseconds
 default_ttl = 300          # Default TTL for entries, in seconds
+default_hash_ttl = 900     # Default TTL for hashes entries, in seconds
```

Comment:
Should we also suggest them to keep this greater than the drainer's loop time (which doesn't seem to be present in config now) so that the value would be in database by the time it is cleared from cache?

Reply:
Currently there is no, interval tick in the drainer's loop and it will constantly keep syncing. But yes when we do add that interval tick we need to have this interval grater than the entire loop. Currently on the safer side I have kept it to 15 minutes. As, it's certain that the drainer will drain all the entries in the database with only db query and network delay to consider.